aten/src/ATen/functorch/BatchRulesReduceOps.cpp

```diff
@@ -55,6 +55,14 @@ Tensor median_decomp(
   return std::get<0>(at::median(self.flatten(), 0, false));
 }
 
+Tensor all_decomp(const Tensor& self) {
+  return at::all(self.flatten(), 0, false);
+}
```

Comment:
Does this remove the `dim` `keepdim` arguments when using `vmap(all)`?

Reply:
This is for the following overload of `all` which is equivalent to doing `all` on a flattened input (similar to `t.sum()`)
https://github.com/pytorch/pytorch/blob/7f2b5ea1e1495a1706ccf88269a0e920354240e3/aten/src/ATen/native/native_functions.yaml#L9399-L9402